server/views/partials/miniProfile/template.njk

```diff
@@ -0,0 +1,29 @@
+<div class="mini-profile govuk-!-margin-top-7" data-qa="mini-profile">
+    <div class="mini-profile-inner">
+        <div class="mini-profile-left">
+            <img class="mini-profile-person-img" src="{{ personImgUrl }}" alt="Image of {{ (person.lastName + ", " + person.firstName) | convertToTitleCase }}" loading="lazy" data-qa="mini-profile-person-img">
+        </div>
+        <div class="mini-profile-right">
+            <ul class="mini-profile-info">
+                <li>
+                    <p class="govuk-heading-s govuk-!-margin-bottom-1">
+                        <a id="mini-profile-link" class="govuk-link--no-visited-state" href="{{ personProfileUrl }}" target="_blank" data-qa="mini-profile-person-profile-link">{{ (person.lastName + ", " + person.firstName) | convertToTitleCase }}</a>
+                    </p>
+                    <span class="govuk-body" data-qa="mini-profile-prisoner-number">{{ person.prisonerNumber }}</span>
+                </li>
+                <li>
+                     <span class="govuk-heading-s govuk-!-margin-bottom-0 govuk-!-padding-0">Date of birth</span>
+                     <span class="govuk-body" data-qa="mini-profile-dob">{{ person.dateOfBirth | formatDate }}</span>
+                </li>
+                <li>
+                    <span class="govuk-heading-s govuk-!-margin-bottom-0 govuk-!-padding-0">Establishment</span>
+                    <span class="govuk-body" data-qa="mini-profile-prison-name">{{ person.prisonName }}</span>
+                </li>
+                <li>
+                    <span class="govuk-heading-s govuk-!-margin-bottom-0 govuk-!-padding-0">Cell number</span>
+                    <span class="govuk-body" data-qa="mini-profile-cell-location">{{ person.cellLocation }}</span>
```

Comment:
I have a feeling this is optional. Might want to check how it's presented, I've seen a few options across different services.

Reply:
I've added an image in the comments showing how its presented. The designs we have are a bit inconsistent - i.e. date formats vary across the same journey, and mini-profile hardly has any info. I'm used some artistic licence to guess at what might be useful, and made things consistent.